src/app/app.js

```diff
@@ -1,4 +1,6 @@
 // Special Debug Console Calls!
+var localStorage = window.localStorage || {};
```

Comment:
add it, that commit is about 3 months old, i'd rather remove the localstorage part than introduce more stuff.